docs/user/library/render/wkt.rst

```diff
@@ -155,3 +155,86 @@ Where *15* indicates the wind intensity and *[kts]* indicates its unit of measur
 And it produces:
 
   .. image:: /images/barbs.png
+  
+WKT Marks as Strokes
+--------------------
+
+The WKT marks are normally converted into images and then repeated along the line geometry,
+in case of continue shapes (e.g a Wave like Shape), the bends across different line segments
+will break the continuity. If the goal is to have the wkt shape repeated and connected
+irrespective of angles between line segments then it is possible to convert the WKT shape
+into a stroke instead of image. 
+
+To enable this make the following vendor option child of <LineSymbolizer> ::
+  
+  <VendorOption name="markAlongLine">true</VendorOption>					
+  ...
+
+(Optional) `markAlongLineScaleLimit` is used define the scale limit.
+
+ Valid values are between 0 and 1
+ 
+ 0 = Always Scale
+ 
+ 1 = Never Scale
+ 
+ Default value = 0.9
+ 
+ This value is used by Stroke to adjust the Shape width to its respective line segment.
+ 
+ 0.9 means that Shape cannot shrink beyond this scale limit to fit on the underlying line segment ::
+ 
+  
+  <VendorOption name="markAlongLineScaleLimit">0.9</VendorOption>
+  ...
+  
+(Optional) `markAlongLineSimplifyTolerance` is used to decide if linear geometry needs to be simplified.
```

Comment:
Odd behavior... I would have used 40% of the mark size as the simplification distance, but always simplify regardless of the short segment count... (well, unless the percentage is zero).
I don't understand how this approach helps honestly, what if one has a bunch of straight lines and then a small accumulation of short segments, wouldn't the line look odd there? 
Suggestion (already made in private): make this the percentage of the mark size, and use it as the simplification distance, e.g., ``(0.4 * 16) = 4.8`` pixels distance.

Reply:
Simplification at higher zoom levels was degrading performance, however I will address the suggested changes.